jgrapht-ext/src/main/java/org/jgrapht/ext/DOTExporter.java

```diff
@@ -42,6 +44,22 @@
      * Default graph id used by the exporter.
      */
     public static final String DEFAULT_GRAPH_ID = "G";
+    /** Keyword for representing strict graphs. */
+    static final String DONT_ALLOW_MULTIPLE_EDGES_KEYWORD = "strict";
+    /** Keyword for directed graphs. */
+    static final String DIRECTED_GRAPH_KEYWORD = "digraph";
+    /** Keyword for undirected graphs. */
+    static final String UNDIRECTED_GRAPH_KEYWORD = "graph";
+    /** Edge operation for directed graphs. */
+    static final String DIRECTED_GRAPH_EDGEOP = "->";
+    /** Edge operation for undirected graphs. */
+    static final String UNDIRECTED_GRAPH_EDGEOP = "--";
```

Comment:
I added them in the `DOTUtils` as a package-private. It will be nice to keep the importer and exporter in sync somehow, at least til it uses antlr.